pkg/kube/client.go

```diff
@@ -520,7 +520,8 @@ func podsReady(pods []api.Pod) bool {
 
 func servicesReady(svc []api.Service) bool {
 	for _, s := range svc {
-		if !api.IsServiceIPSet(&s) {
+		// Make sure the service is not explicitly set to "None" before checking the IP
+		if s.Spec.ClusterIP != api.ClusterIPNone && !api.IsServiceIPSet(&s) {
```

Comment:
It is actually where I stole the logic from 😄 . But that function will return `false` if the `clusterIP` is set to `None`, which is not what we want